os_migrate/roles/conversion_host/tasks/conv_hosts_inventory.yml

```diff
@@ -26,22 +26,30 @@
       os_migrate_conversion_ca_cert: "{{ os_migrate_dst_ca_cert|default(omit) }}"
       os_migrate_conversion_client_cert: "{{ os_migrate_dst_client_cert|default(omit) }}"
       os_migrate_conversion_client_key: "{{ os_migrate_dst_client_key|default(omit) }}"
+  when:
+    - item["_migration_params"]["data_copy"]|default(true)
```

Comment:
The `item` here is a conversion host. It will not have `_migration_params` on it like the workloads do, so i think what's happening here is that `default(true)` gets used always. We will have to think of some other solution. Maybe we should not include this file at all if `os_migrate_workloads_data_copy` is `false`?